src/features/metadata-instance-editor/Instance.js

```diff
@@ -549,10 +551,10 @@ class Instance extends React.PureComponent<Props, State> {
         return this.canEdit() && isEditing;
     }
 
-    renderEditButton = () => {
+    renderEditButton = (canEditInstance: boolean) => {
         const { intl, isDirty }: Props = this.props;
         const { isBusy }: State = this.state;
-        const canEdit = this.canEdit();
+        const canEdit = this.canEdit() && canEditInstance;
```

Comment:
based on the method descriptions of `canEdit` and `isEditing`, the button only renders now when it's an editing mode, is that correct?
(since `canEditInstance = isEditing && hasCascadePermission`)

how does the component get into editing mode now?